modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/services/discovery/impl/DiscoveryServiceImpl.java

```diff
@@ -48,9 +48,9 @@
 public class DiscoveryServiceImpl implements DiscoveryService {
 
     public static final String FEDERATED = "Federated";
+    private static final String DELIMITER = "@";
+    private static final int DOMAIN_SEGMENT = 1;
     private static Log log = LogFactory.getLog(DiscoveryServiceImpl.class);
-    @Resource
-    private WebServiceContext context;
 
     /**
      * This method returns the OnPremise AuthPolicy and next two endpoint the mobile device should
```

Comment:
unless it doesn't have return type,This method used to set  three type of response values through Holder<DiscoveryResponse> response,thats why I commented as it was.